### PR TITLE
Restructure SysEleven Stack documentation

### PR DESCRIPTION
**Pull request by stefreak**
Based on https://www.divio.com/blog/documentation/

### CODE REVIEW

File: pages/01.syseleven-stack/default.de.md [hunk 1]
```diff
@@ -12,14 +12,29 @@ page-toc:
...
 
 ## Support
 
 Als der Hersteller des [SysEleven Stacks](https://dashboard.cloud.syseleven.net/) bieten wir **kostenlosen Support** für unsere Cloud-Kunden.  
-Weitere Informationen finden Sie auf unserer [Support Seite](../04.Support/default.de.md).
+Weitere Informationen finden Sie auf unserer [Support Seite](../06.Support/default.de.md).
+
+## Beiträge editieren
+
+Unsere [Dokumentation ist auf GitHub](https://github.com/syseleven/grav-docu-syseleven-stack) und wir freuen uns auf Änderungswünsche in Form von Pull Requests.
```
Comment (stefreak) [inline pages/01.syseleven-stack/default.de.md]:
```suggestion
Unsere [Dokumentation ist auf GitHub](https://github.com/syseleven/grav-docu-syseleven-stack) und wir freuen uns über Änderungswünsche in Form von Pull Requests.
```

File: pages/01.syseleven-stack/default.de.md [hunk 2]
```diff
@@ -12,14 +12,29 @@ page-toc:
...
 
-["Erste Schritte"](../03.Tutorials/02.firststeps/docs.en.md) zeigt, wie über das Dashboard des SysEleven Stack eine virtuelle Maschine gestartet werden kann.  
+["Erste Schritte"](../02.Tutorials/02.firststeps/docs.en.md) zeigt, wie über das Dashboard des SysEleven Stack eine virtuelle Maschine gestartet werden kann.  
 Weitere Beispiele zeigen, wie über die API des SysEleven Stack komplexere Setups verwaltet werden können.
 
-## Komplexe Setups
+## Inhalt
 
-Auf dem Einstieg aufbauend stellt SysEleven eine [Bibliothek an Heat-Templates](https://github.com/syseleven/heat-examples) bereit, die nach dem Einstieg den Aufbau komplexerer Setups ermöglichen.
+Unsere Dokmentation ist aufgeteilt in die folgenden Sektionen:
```
Comment (bpetermannS11) [inline pages/01.syseleven-stack/default.de.md]:
```suggestion
Unsere Dokumentation ist aufgeteilt in die folgenden Sektionen:
```

File: pages/01.syseleven-stack/default.en.md [hunk 3]
```diff
@@ -12,12 +12,27 @@ page-toc:
...
+## Contents
 
-On top of the first step examples, SysEleven provides a [library of heat templates](https://github.com/syseleven/heat-examples) that will help you with the creation of a complex setup.
+Our documentation is structured in four distinct sections:
+
+| Section             | Description |
+| -------------------:| -------------|
+| [Tutorials](../02.Tutorials/docs.en.md)            | Learning-oriented step-by-step guides, to get started with SysEleven Stack. |
+| [How-to guides](../03.Howtos/docs.en.md)           | Goal-oriented step-by-step guides, for solving specific problems. |
+| [Reference articles](../04.Reference/docs.en.md)   | Detailled description of all the services we are offering in the context of SysEleven Stack. |
```
Comment (bpetermannS11) [inline pages/01.syseleven-stack/default.en.md]:
```suggestion
| [Reference articles](../04.Reference/docs.en.md)   | Detailed description of all the services we are offering in the context of SysEleven Stack. |
```

File: pages/02.Tutorials/docs.en.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+---
+title: Tutorials
+page-toc:
+    active: false
+taxonomy:
+    category:
+        - tutorials
```
Comment (bpetermannS11) [inline pages/02.Tutorials/docs.en.md]:
```suggestion
        - docs
```
All other taxonomy category values are "docs" too.
Or rename the category of the german variant to "tutorials" instead.

File: pages/03.Howtos/docs.de.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+title: How-to guides
+page-toc:
+    active: false
+taxonomy:
+    category:
+        - how-to
```
Comment (bpetermannS11) [inline pages/03.Howtos/docs.de.md]: Shouldn't it be the same category as the english variant?
I saw some other category name "howtos". Should be consistent.

Comment (stefreak) [reply pages/03.Howtos/docs.de.md]:
```suggestion
        - docs
```

**Pull request merged**